src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_IUsingStatement.cs

```diff
@@ -1962,43 +1955,22 @@ class NotDisposable
                         IParameterReferenceOperation: b (OperationKind.ParameterReference, Type: System.Boolean) (Syntax: 'b')
                       Right: 
                         ILiteralOperation (OperationKind.Literal, Type: System.Boolean, Constant: True) (Syntax: 'true')
-
-            Next (Regular) Block[B8]
+           Next (Regular) Block[B6]
                 Finalizing: {R5}
                 Leaving: {R4} {R3} {R1}
     }
     .finally {R5}
     {
         Block[B5] - Block
             Predecessors (0)
-            Statements (0)
-            Jump if True (Regular) to Block[B7]
-                IIsNullOperation (OperationKind.IsNull, Type: System.Boolean, IsInvalid, IsImplicit) (Syntax: 'GetDisposable() ?? input')
```

Comment:
>IIsNullOperation (OperationKind.IsNull, Type: System.Boolean, IsInvalid, IsImplicit) (Syntax: 'GetDisposable() ?? input') [](start = 16, length = 121)

It appears that there are no changes like that for VB tests. Are we not testing similar error scenarios for VB?

Reply:
There were no tests, I've added a couple to cover the scenarios and ensure they are as expected.